### PR TITLE
#55 Update quick start VM docs for oharastream 0.11.0 release

### PR DESCRIPTION
**Pull request by konekoya**
TODOS:

- [x] Update release screenshot
- [x] Update VM loading screenshots

### CODE REVIEW

File: content/en/docs/master/quickstart/quickstartvm.md [hunk 1]
```diff
@@ -16,22 +16,22 @@ weight: 1
...
+  An OVA (Open Virtual Appliance) file, a pre-prepared virtual machine image for Ohara quickstart.
   You can download the image file (.ova) from the [release](https://github.com/oharastream/ohara-quickstart/releases/) page.
 
-  {{< figure src="../img/download_assets.png" title="Assets of a release" lightbox="true" >}}
+  {{< figure src="../img/download_assets.png" alt="Download asserts" alt="Release asserts" lightbox="true" >}}
 
-{{% alert tip %}} 
-Please download the VirtualBox from [here](https://www.virtualbox.org/wiki/Downloads), 
-and reference [this article](https://www.virtualbox.org/manual/ch02.html) on how to install it on your machine. 
+{{% alert note %}}
```
Comment (konekoya) [inline content/en/docs/master/quickstart/quickstartvm.md:27]: It looks like our them only support `note` and `warning` alerts...

File: content/en/docs/master/quickstart/quickstartvm.md [hunk 2]
```diff
@@ -296,219 +292,215 @@ You can also delete a vertex by double clicking on it.
...
 
-{{< figure src="../img/add-stream-name.png" title="Add stream name" lightbox="true" height="100%" width="100%" >}}
+{{< figure src="../img/add-stream-name.png" alt="Add stream name" lightbox="true" height="100%" width="100%" >}}
 
 - Now let's work through the configuration together. Hover over the stream component and click on the "Configure" button to open the configure dialog
 - Fill out the form with the following settings and click the "**SAVE CHANGES**" button:
-  - The filtered value: _M_
-  - The filtered header name: _Sex_
+  - header name to be filtered: _"Sex"_
+  - column value to be filtered: _"M"_
```
Comment (konekoya) [inline content/en/docs/master/quickstart/quickstartvm.md:383]: The order was changed

File: content/en/docs/master/quickstart/quickstartvm.md (hunk 1 above)
Comment (vitojeng) [reply content/en/docs/master/quickstart/quickstartvm.md:27]: IIRC, the difference is tip without any border, just only prepend an "i" icon

Comment (konekoya) [reply content/en/docs/master/quickstart/quickstartvm.md:27]: Yep, but the color for border and background are the same just like `note`... or I can tweak the style and add one with green border and background just like our previous docs 🤔

Comment (vitojeng) [reply content/en/docs/master/quickstart/quickstartvm.md:27]: I just write the difference between "tip" and "note". :)
You can make any changes according to your own ideas.

**Comment by vitojeng** [comment]
Thanks @konekoya !

**Pull request merged**